scripts/pipelines/github/templates/test/test-pipeline.yml.template

```
@@ -0,0 +1,48 @@
+name: $pipelineName
+
+on:
+  workflow_run:
+    workflows: $buildPipelineName
+    types: completed
+  workflow_dispatch:
+# mark to insert additional artifact input #
+
+permissions:
+  actions: write
+
+env:
+  build_pipeline: 'build-pipeline.yml'
```

```suggestion
  build_pipeline: 'build-pipeline.yml'
  ciPipelineYamlName: 'ci-pipeline.yml'
```

for the download part